core/camel-support/src/main/java/org/apache/camel/support/EventHelper.java

```diff
@@ -1300,7 +1301,7 @@ public static boolean notifyStepFailed(CamelContext context, Exchange exchange,
 
     private static boolean doNotifyEvent(EventNotifier notifier, CamelEvent event) {
         // only notify if notifier is started
-        boolean started = true;
+        started = true;
```

Comment:
remove started = true as we need to check the notifier instance.

And if its NOT a stateful service then we can regard it as started